x-pack/plugins/index_lifecycle_management/public/extend_index_management/index.tsx

```diff
@@ -35,22 +38,22 @@ export const retryLifecycleActionExtension = ({ indices }) => {
       'xpack.indexLifecycleMgmt.retryIndexLifecycleAction.retriedLifecycleMessage',
       {
         defaultMessage: 'Called retry lifecycle step for: {indexNames}',
-        values: { indexNames: indexNames.map((indexName) => `"${indexName}"`).join(', ') },
+        values: { indexNames: indexNames.map((indexName: string) => `"${indexName}"`).join(', ') },
       }
     ),
   };
 };
 
-export const removeLifecyclePolicyActionExtension = ({ indices, reloadIndices }) => {
+export const removeLifecyclePolicyActionExtension = ({ indices, reloadIndices }: any) => {
```

Comment:
At the very least, I think this could be improved to:

```suggestion
export const removeLifecyclePolicyActionExtension = ({ indices, reloadIndices }: {indices: any; reloadIndices: () => void}) => {
```